### DISCUSSION TITLE
Usage of special quote block in readme

### DISCUSSION THREAD

**Discussion by Andre601** [giscus/giscus#543] [Ideas]
Kind of a pointless "idea", but I want to mention it anyways.

GitHub (secretly) added a feature to display a special blockquote when using `**Note**` or `**Warning**` as the first line.

For example:
```markdown
> **Note**
> Giscus is still under active development. GitHub is also still actively developing Discussions and its API. Thus some features of giscus may break or change over time.
```
becomes

> **Note**
> Giscus is still under active development. GitHub is also still actively developing Discussions and its API. Thus some features of giscus may break or change over time.

As I said is this not really a useful or important change, but it could make the readme look slightly better?

**Comment by laymonage** [comment]
Thanks, added in #544 but I've left the README in other languages as they are until GitHub adds support for custom text.
